tests/unit/select/index.html

```diff
@@ -43,6 +43,25 @@
 			<option value="4">The 4th Option</option>
 			</select>
 		</div>
+		<div id="width-test-container">
+			<div data-role="fieldcontain">
+				<label for="width-test">Multiple:</label>
+				<select name="width-test" id="width-test" multiple="multiple" data-inline="true" data-nstest-native-menu="false" data-nstest-overlay-theme="b" data-nstest-theme="x">
```

Comment:
yes any where we use it this should be fixed. that would be a separate PR though.